Exception 5 thrown!
5

==========
Statistics
==========

Objects created:
   Main |-> 1
   Exception |-> 1

Methods invoked:
   Main.Main |-> 1
   Main.foo |-> 1
   Exception.Exception |-> 1
   Exception.get |-> 1
